virttest/utils_conn.py

```diff
@@ -603,6 +603,8 @@ def conn_setup(self):
         ssh_copy_id = self.SSH_COPY_ID
         ssh_agent = self.SSH_AGENT
         shell = self.SHELL
+        if rTimeout < 0:
+            rTimeout = 10
```

Comment:
This judgement make no sense, timeout could be None. 
Just pass it to handle_prompts is enough.